georaster-layer-for-leaflet.js

```diff
@@ -56,8 +56,58 @@ var GeoRasterLayer = L.GridLayer.extend({
             console.error("ERROR initializing GeoTIFFLayer", error);
         }
     },
+    
+    getRasters: async function(
+          map, tileNwPoint, height_of_rectangle_in_pixels, width_of_rectangle_in_pixels,
+          coords, pixelHeight, pixelWidth, number_of_rectangles_across, number_of_rectangles_down, ymax, xmin) {
+      // called if georaster was constructed from URL and we need to get
+      // data separately for each tile
+      // aka "COG mode"
+
+      const raster_coords_for_tile_coords = function(h, w) {
+        let y_center_in_map_pixels = tileNwPoint.y + (h + 0.5) * height_of_rectangle_in_pixels;
+        let latWestPoint = L.point(tileNwPoint.x, y_center_in_map_pixels);
+        let latWest = map.unproject(latWestPoint, coords.z);
+        let lat = latWest.lat;
```

Comment:
Totally not a big deal, but you could do `let { lat } = latWest;` if you think it looks better.